Guitars_2019-08-05.sql

```diff
@@ -0,0 +1,158 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 192.168.20.20 (MySQL 5.6.44)
+# Database: Guitars
+# Generation Time: 2019-08-05 10:18:36 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table brand
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `brand`;
+
+CREATE TABLE `brand` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `brand` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `brand` WRITE;
+/*!40000 ALTER TABLE `brand` DISABLE KEYS */;
+
+INSERT INTO `brand` (`id`, `brand`)
+VALUES
+	(1,'Fender'),
+	(2,'Gibson'),
+	(3,'Epiphone'),
+	(4,'Squier'),
+	(5,'Ibanez'),
+	(6,'PRS'),
+	(7,'Yamaha');
+
+/*!40000 ALTER TABLE `brand` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table countries
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `countries`;
+
+CREATE TABLE `countries` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `country` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `countries` WRITE;
+/*!40000 ALTER TABLE `countries` DISABLE KEYS */;
+
+INSERT INTO `countries` (`id`, `country`)
+VALUES
+	(1,'America'),
+	(2,'Mexico'),
+	(3,'Japan'),
+	(4,'Scotland'),
+	(5,'Spain'),
+	(6,'China');
+
+/*!40000 ALTER TABLE `countries` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table guitars
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `guitars`;
+
+CREATE TABLE `guitars` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `brand` int(11) unsigned NOT NULL,
+  `model` varchar(255) NOT NULL DEFAULT '',
+  `year` year(4) DEFAULT NULL,
+  `type` int(11) unsigned NOT NULL,
+  `country` int(11) unsigned NOT NULL,
+  `LH or RH` enum('LH','RH') NOT NULL,
+  `value` int(11) DEFAULT NULL,
+  `serialCode` int(11) DEFAULT NULL,
```

Comment:
Are serial codes just numbers? Are they never ever going to contain any letters?